lib/ansible/galaxy/api.py

```diff
@@ -330,10 +330,7 @@ def __lt__(self, other_galaxy_api):
         if not isinstance(other_galaxy_api, self.__class__):
             return NotImplemented
 
-        return (
-            self._priority > other_galaxy_api._priority or
-            self.name < self.name
-        )
+        return self._priority > other_galaxy_api._priority
```

Comment:
Looks good to me. Since `or False` didn't contribute to the overall logic of the conditional statement, does this even need a changelog?